### DISCUSSION TITLE
Inconsistent approximation default for Auto ARIMA

### DISCUSSION THREAD

**Discussion by alech97** [Nixtla/statsforecast#59] [Q&A] [answered]
https://github.com/Nixtla/statsforecast/blob/b4b59c903c8543b1290338033f6becfa74e985a5/statsforecast/models.py#L238

**approximation** is set to **False** by default in auto_arima(), but auto_arima_f()'s default handles this logic similarly to R's auto.arima() already:

https://github.com/Nixtla/statsforecast/blob/b4b59c903c8543b1290338033f6becfa74e985a5/statsforecast/arima.py#L1438

**Comment by AzulGarza** [comment, marked as answer]
Hi @alech97! We decided to set `approximation = False` to be comparable with `pmdarima`. That package (if I remember correctly) doesn't have the option to use approximation.  Even without approximation, our implementation is a lot faster. 😄 
The use of `approximation=True` reduces the time even more, but may also reduce the accuracy.

**Comment by mergenthaler** [comment]
@all-contributors please add @alech97 for question

**Comment by allcontributors** [comment]
@mergenthaler 

I've put up [a pull request](https://github.com/Nixtla/statsforecast/pull/42) to add @alech97! :tada:

**Comment by MarcoGorelli** [comment]
> The use of approximation=False reduces the time even more, but may also reduce the accuracy.

I presume you meant "The use of approximation=True"?

**Comment by AzulGarza** [comment]
Yes, @MarcoGorelli that is correct! Thanks for catching that :)
